i3-input/main.c

```diff
@@ -386,10 +394,15 @@ int main(int argc, char *argv[]) {
         {"prefix", required_argument, 0, 'p'},
         {"format", required_argument, 0, 'F'},
         {"font", required_argument, 0, 'f'},
+        {"fg-color", required_argument, 0, 'C'},
```

Comment:
i3lock uses lower-case 'c', maybe have some consistency since it's available? https://github.com/i3/i3lock/blob/f3b061233e28ad0cf920c9421adc2459b6920812/i3lock.c#L1025

```suggestion
        {"fg-color", required_argument, 0, 'c'},
```

Reply:
Or even remove the short options since they don't generally make much sense.